TOF powder diffraction GUI (#33820)
mantidproject/mantid: docs/source/interfaces/direct/dns_powder_tof/dns_powder_tof_plotting_tab.rst

.. _dns_powder_tof_plotting_tab-ref:

DNS Powder TOF Plotting Tab
---------------------------

.. image::  ../../../images/DNS_interface_powder_tof_plotting_tab.png
   :align: center
   :height: 400px
\

> The plotting tab of the powder TOF mode, is just the Mantid :ref:`sliceviewer`.

Reviewer: ```suggestion
For usage of the Powder TOF plotting tab see usage for the mantid :ref:`sliceviewer`.
```
